build.gradle

```diff
@@ -88,6 +88,7 @@ configurations.all {
         force "com.fasterxml.woodstox:woodstox-core:6.4.0"
         force "org.scala-lang:scala-library:2.13.9"
         force "org.apache.bcel:bcel:6.6.0" // This line should be removed once Spotbugs is upgraded to 4.7.4
+        force "org.xerial.snappy:snappy-java:1.1.10.5"
```

Comment:
@jaguilar-atl why do we need force here? are there some transitive deps?

Reply:
+1